### PR TITLE
Isolate puppet_helpers workaround for windows os family

### PR DESCRIPTION
**Pull request by carabasdaniel**
Closes: https://github.com/puppetlabs/puppet_litmus/pull/377

### CODE REVIEW

**Comment by carabasdaniel** [comment]
Tests here: https://github.com/puppetlabs/puppetlabs-concat/pull/684 and https://github.com/puppetlabs/puppetlabs-mysql/pull/1362

**Comment by sheenaajay** [comment]
Tested locally
Screenshot 2021-01-29 at 09 59 23
Screenshot 2021-01-29 at 09 59 32

**Comment by sheenaajay** [comment]
looks like https://github.com/puppetlabs/puppetlabs-concat/pull/684/checks?check_run_id=1785454021#step:13:112 concat modules  failures are specific to module.

**Review by sheenaajay — APPROVED** [review]

File: lib/puppet_litmus/puppet_helpers.rb [hunk 1]
```diff
@@ -83,18 +83,20 @@ def apply_manifest(manifest, opts = {})
       command_to_run += ' --noop' if !opts[:noop].nil? && (opts[:noop] == true)
       command_to_run += ' --detailed-exitcodes' if use_detailed_exit_codes == true
 
-      command_to_run = "try { #{command_to_run}; exit $LASTEXITCODE } catch { write-error $_ ; exit 1 }" if os[:family] == 'windows'
-
       span.add_field('litmus.command_to_run', command_to_run)
```
Comment (DavidS) [inline lib/puppet_litmus/puppet_helpers.rb:86]: not capturing the actual command ran on windows, anymore.

Comment (carabasdaniel) [reply lib/puppet_litmus/puppet_helpers.rb]: Moved the span inside the conditions to capture the commands

**Comment by carabasdaniel** [comment]
Closes: https://github.com/puppetlabs/puppetlabs-concat/pull/684

**Review by DavidS — APPROVED** [review]

**Pull request merged**